src/js/fc.js

```diff
@@ -1,23 +1,105 @@
 'use strict';
 
+const INITIAL_CONFIG = {
```

Comment:
The `fc.js` contains the data retrieved/sent from/to the FC using the MSP protocol. 

The major part of this variables seems that are temporal variables to do some calculations at some parts of the code, so they must not be a global variable in this file. Other are statistics or calculated things that go into the Configurator, in this case they must go into another file or not to be globals. Too much of them to look one by one at this moment of the POC.

Reply:
Sorry, I've seen they were used in the reset state of this file, you only changed them of place... I'm not too sure if this is the correct way of reset things, but this is another question for another PR, no Vue related....